pkg/reconciler/openshift/tektonconfig/extension.go

```diff
@@ -94,6 +94,13 @@ func (oe openshiftExtension) PostReconcile(ctx context.Context, comp v1alpha1.Te
 			return err
 		}
 	}
+
+	if configInstance.Spec.Profile == common.ProfileBasic {
+		addon, _ := extension.GetAddon(oe.operatorClientSet.OperatorV1alpha1().TektonAddons(), common.DashboardResourceName)
```

Comment:
this err should also be checked, isn't it :)

Reply:
yes 👍🏻